server/src/user/user.controller.ts

```diff
@@ -1,15 +1,34 @@
-import { Controller, Get, UseGuards } from '@nestjs/common';
+import { Body, Controller, Get, Put, Res, UseGuards } from '@nestjs/common';
 import { UserService } from './user.service';
 import { AuthGuard } from '../auth/guard';
 import { GetUser } from '../auth/decorator';
+import { UpdateUserDto } from './dto/update-user-dto';
+import { Response } from 'express';
 
+@UseGuards(AuthGuard)
 @Controller('me')
 export class UserController {
   constructor(private userService: UserService) {}
 
-  @UseGuards(AuthGuard)
   @Get()
   getMe(@GetUser('id') userId: number) {
     return this.userService.getMe(userId);
   }
+
+  @Put(':userId')
```

Comment:
You don't need to pass `:userId` because you can get the data using `@GetUser()` decorator.  Simply `/PUT/me` is fine.
 In `/auth/strategy/jwt.stragetgy.ts`, we check token in cookie and find user from the database!

Reply:
I see! I fixed both client and server side. 
342d09ff8aa72a79e494a1b985d1ad5f95a9c9a7
2c83d9aeb259db6607a5a82957c69a6df321411d